build-system/server.js

```diff
@@ -346,16 +346,19 @@ function liveListTombstone(liveList) {
   }
 }
 
+
+// Value is inclusive of both min and max values.
+function range(min, max) {
```

Comment:
Please briefly document this method. Not your fault but the implementation is pretty inscrutable.